src/convertToObject.js

```diff
@@ -6,7 +6,25 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = {};
+  const lines = sourceString.replace('\n', '').split(';');
+
+  for (let line of lines) {
+    line = line.trim();
```

Comment:
why?
if u want to do this - create a separate const variable for this
```suggestion
  for (const line of lines) {
    const trimmedLine = line.trim();
```